### PR TITLE
Issues 370 526 579 instances

### PR DESCRIPTION
**Pull request by uscholdm**
- Updated annotations for coherent unit classes, per issue [#146]
- Added definitions for unit of measure instances per issue [#526] 
- Added unit symbols for unit instances instances per issue [#579]

### CODE REVIEW

File: docs/ReleaseNotes.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 gist Release Notes
 =====
 
+### Patch Updates
+
+- Updated annotations for coherent unit classes, per issue [#146](https://github.com/semanticarts/gist/issues/146)
+- Added definitions for unit of measure instances per issue [#526] https://github.com/semanticarts/gist/issues/526
```
Comment (rjyounes) [inline docs/ReleaseNotes.md:7]: Need parens around the URL

File: docs/ReleaseNotes.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
 gist Release Notes
 =====
 
+### Patch Updates
+
+- Updated annotations for coherent unit classes, per issue [#146](https://github.com/semanticarts/gist/issues/146)
+- Added definitions for unit of measure instances per issue [#526] https://github.com/semanticarts/gist/issues/526 
+- Added unit symbols for unit instances instances per issue [#579] https://github.com/semanticarts/gist/issues/579
```
Comment (rjyounes) [inline docs/ReleaseNotes.md:8]: Need parens around the URL

File: gistCore.ttl [hunk 3]
```diff
@@ -324,7 +326,8 @@ gist:CoherentUnit
 	skos:prefLabel "Coherent Unit"^^xsd:string ;
 	skos:scopeNote
 		"Coherent unit is the physics term for this, informally you might think of it as the standard unit for a given dimension."^^xsd:string ,
-		"In principle, the CoherentUnit for a ProductUnit or RatioUnit can be inferred by recursively decomposing the products and ratios into their respective CoherentUnits, bottoming out in SimpleUnits"^^xsd:string
+		"In principle, the CoherentUnit for a ProductUnit or RatioUnit can be inferred by recursively decomposing the products and ratios into their respective CoherentUnits, bottoming out in SimpleUnits"^^xsd:string ,
+		"See: http://www.eim.gr/metrology/measurement-units/si-derived-units/coherent-derived-units/"^^xsd:string
```
Comment (rjyounes) [inline gistCore.ttl:330]: How about putting the "See" statement in an `rdfs:seeAlso`?

File: gistCore.ttl [hunk 4]
```diff
@@ -2654,7 +2663,9 @@ gist:_bit
 		owl:Thing ,
 		gist:BaseUnit
 		;
-	skos:definition "A bit (short for binary digit) is the smallest unit of data in a computer. A bit has a single binary value, either 0 or 1."^^xsd:string ;
+	skos:definition "The base unit for measuring information."^^xsd:string ;
```
Comment (rjyounes) [inline gistCore.ttl:2666]: I would say more specifically "digital information."

File: gistCore.ttl [hunk 5]
```diff
@@ -2702,9 +2722,12 @@ gist:_kelvin
 		owl:Thing ,
 		gist:BaseUnit
 		;
+	skos:definition "The base unit for measuring temperature."^^xsd:string ;
+	skos:prefLabel "Kelvin"^^xsd:string ;
```
Comment (rjyounes) [inline gistCore.ttl:2726]: Should the prefLabel here be lowercased, like the others?

File: gistCore.ttl [hunk 6]
```diff
@@ -2761,6 +2799,9 @@ gist:_one_day
 		owl:Thing ,
 		gist:Duration
 		;
+	skos:definition "A duration equal to 24 hours."^^xsd:string ;
+	skos:prefLabel "one day"^^xsd:string ;
+	skos:scopeNote "This is a gist:Magnutude, not a unit. Semantically it is the same as the unit, day."^^xsd:string ;
```
Comment (rjyounes) [inline gistCore.ttl:2804]: Cheeky. Though you know I agree, I wouldn't assert semantic equivalence until the team has decided to collapse the magnitudes and units generally.

File: gistCore.ttl [hunk 7]
```diff
@@ -2771,6 +2812,9 @@ gist:_one_millisecond
 		owl:Thing ,
 		gist:Duration
 		;
+	skos:definition "A duration equal to one millisecond."^^xsd:string ;
+	skos:prefLabel "one millisecond"^^xsd:string ;
+	skos:scopeNote "This is a gist:Magnutude, not a unit. Semantically it is the same as the unit, millisecond."^^xsd:string ;
```
Comment (rjyounes) [inline gistCore.ttl:2817]: Same as above

File: gistCore.ttl [hunk 8]
```diff
@@ -2781,6 +2825,9 @@ gist:_one_minute
 		owl:Thing ,
 		gist:Duration
 		;
+	skos:definition "A duration equal to one minute."^^xsd:string ;
+	skos:prefLabel "one minute"^^xsd:string ;
+	skos:scopeNote "This is a gist:Magnutude, not a unit. Semantically it is the same as the unit, minute."^^xsd:string ;
```
Comment (rjyounes) [inline gistCore.ttl:2830]: Same as above

File: gistCore.ttl [hunk 9]
```diff
@@ -2791,8 +2838,11 @@ gist:_second
 		owl:Thing ,
```
Comment (rjyounes) [inline gistCore.ttl:2838]: Why do we have these `owl:Thing` assertions everywhere? Does the serializer add them, or are they leftover. I recommend removing them.

**Review by rjyounes — CHANGES_REQUESTED** [review]
Looks good overall. I've made a few suggestions.

File: docs/ReleaseNotes.md (hunk 1 above)
Comment (uscholdm) [reply docs/ReleaseNotes.md:7]: ok

File: gistCore.ttl (hunk 3 above)
Comment (uscholdm) [reply gistCore.ttl:330]: Not a bad idea. There is exactly one other use of `seeAlso`, and no other uses of `scopeNote` that ought to be `seeAlso` instead.  This should be in the style guide, if it isn't already.

File: gistCore.ttl (hunk 4 above)
Comment (uscholdm) [reply gistCore.ttl:2666]: ok

File: gistCore.ttl (hunk 5 above)
Comment (uscholdm) [reply gistCore.ttl:2726]: I wondered about that. My thinking was that unlike all the the others, (k)Kelvin is pretty much always capitalized - perhaps due to its originally being a person's name.  It seemed appropriate for a label, it looked wrong in low case.

File: gistCore.ttl (hunk 6 above)
Comment (uscholdm) [reply gistCore.ttl:2804]: I don't mean the formal semantics of OWL, I mean what it is representing in the world. I gave it a slight rewording. 
- `This is a gist:Magnutude, not a unit of measure, it means essentially the same thing as the unit, day.`

File: gistCore.ttl (hunk 9 above)
Comment (uscholdm) [reply gistCore.ttl:2838]: I think an artifact of Protege, but this is a separate issue.

**Review by rjyounes — APPROVED** [review]

**Pull request closed without merging**